zerver/lib/onboarding.py

```diff
@@ -172,12 +172,9 @@ def select_welcome_bot_response(human_response_lower: str) -> str:
     elif human_response_lower in ["stream", "streams", "channel", "channels"]:
         return "".join(
             [
-                _(
-                    "In Zulip, streams [determine who gets a message](/help/streams-and-topics). "
-                    "They are similar to channels in other chat apps."
-                )
+                _("In Zulip, channels [determine who gets a message](/help/streams-and-topics).")
                 + "\n\n",
-                _("[Browse and subscribe to streams](#streams/all)."),
+                _("[Browse and subscribe to channels](#streams/all)."),
             ]
```

Comment:
We'll possibly end up skipping this commit, since these are being wholesale replaced in https://github.com/zulip/zulip/issues/29297; I'm just not sure which will merge first.

Reply:
This commit is still included in these changes for now and I'm following the work on the issue noted above so that I can coordinate this renaming with that update.